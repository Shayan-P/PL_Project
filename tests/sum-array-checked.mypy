checked

a = [1,2,3];
b = 3;

print(a + a);
print(b + b);
