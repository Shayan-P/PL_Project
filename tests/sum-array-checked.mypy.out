type-checked!
'(1 2 3 1 2 3)
6
'finished
